lib/src/fortio.cpp

```diff
@@ -0,0 +1,384 @@
+#include <algorithm>
+#include <ciso646>
+#include <cstdint>
+#include <cstdio>
+#include <cstring>
+#include <limits>
+#include <stdexcept>
+
+#include <ecl/fortio.h>
+
+#ifdef HAVE_NETINET_IN_H
```

Comment:
Depending a bit on how things are done; but I think it will be necessary to do an endian swapping of the integer element from the header block; that would require access to the `htonl()` function also in another compilation unit. Would be good if this `#ifdef ... #endif` block was assembled in a private header.

Reply:
I think you're right, but it's a bit premature to re-introduce a config.h (or similar) with only one module implemented. Mind if we wait a bit?

Reply:
> Mind if we wait a bit?

Sure.